xml2rfc/writers/preptool.py

```diff
@@ -510,16 +510,22 @@ def check_ascii_text(self, e, p):
                     if not c.get('ascii') and not c.tag in bare_unicode_tags and not is_script(c.text, 'Latin'):
                         self.err(c, 'Found non-ascii content without matching ascii attribute in <%s>: %s' % (c.tag, show))
                 else:
-                    self.err(c, 'Found non-ascii characters outside of elements that can have non-ascii content, in <%s>: %s' % (c.tag, show))
-                    c.text = downcode(c.text)
+                    if self.draft:
+                        self.err(c, 'Found non-ascii characters outside of elements that can have non-ascii content, in <%s>: %s' % (c.tag, show))
+                        c.text = downcode(c.text)
+                    else:
+                        self.warn(c, 'Found non-ascii characters outside of elements that can have non-ascii content, in <%s>: %s' % (c.tag, show))
```

Comment:
`xml2rfc` will not hit this warning for document with RFC number that has Unicode content on `<t>` because `enable_rfc_mode_rfc7997_treatment()` adds `<t>` to `bare_unicode_tags` and `unicode_content_tags`.

Reply:
Correct.  The change directly to bare_unicode_tags and unicode_content_tags probably obviates making the change to preptool.py.  I was hoping that the prep tool change was sufficient, also because of the convenient warning, but xml2rfc then simply downcodes the text, which is not useful.